aqua-examples/near-integration/README.md

```diff
@@ -338,80 +451,77 @@ data Result:
     stderr: string
     stdout: string
 
-service RPCService:
-    node_status(network_id: string, block_ref:string) -> Result
+service NearRpcServices:
+    node_status(network_id: string) -> Result
 
-func rpc_foo(network_id: string, block_ref:string, node_string, service_id: string ) -> string: 
+func rpc_foo(network_id: string, node: string, service_id: string) -> string: 
     on node:
-        RPCService service_id
-        res <- RPCService.node_status(network_id, block_ref)
+        NearRpcServices service_id
+        res <- NearRpcServices.node_status(network_id)
         if res.stderr:
             result <<- "call failed"
         else:
             result <<- res.stdout
         <- result
 ```
 
-Before we can use our Fluence NEAR adapter, we need to deploy our Wasm modules to one or more host peers. We can do that with [Aqua CLI](https://doc.fluence.dev/aqua-book/aqua-cli):
+Before we can use our Fluence NEAR adapter, we need to deploy our Wasm modules to one or more host peers. We can do that with [Fluence CLI](https://github.com/fluencelabs/fluence-cli#fluence-deploy):
 
 ```bash
-aqua remote deploy_service \
-     --addr /dns4/kras-04.fluence.dev/tcp/19001/wss/p2p/12D3KooWFEwNWcHqi9rtsmDhsYcDbRUCDXH84RC4FW6UfsFWaoHi \
-     --config-path configs/near_deploy_cfg.json \
-     --service near-adapter
+fluence deploy
```

Comment:
i think there is a step missing: `? Do you want to deploy all of these services? Yes`

Reply:
yep, I need to update the output as the question was introduced recently. thanks!